modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -101,28 +91,47 @@ const setupiOSOverrides = (editor: Editor) => {
   };
 };
 
-const show = (editor: Editor, e: EditorEvent<TouchEvent>, items: MenuItems, backstage: UiFactoryBackstage, contextmenu: AlloyComponent, useNodeAnchor: boolean, highlightImmediately: boolean) => {
-  const anchorSpec = useNodeAnchor ? getNodeAnchor(editor) : getPointAnchorSpec(editor, e);
+const getAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>, anchorType: Coords.ContextMenuAnchorType) => {
+  if (anchorType === 'point') {
+    return {
+      bubble: Bubble.nu(0, bubbleSize, bubbleAlignments),
+      layouts,
+      overrides: {
+        maxWidthFunction: MaxWidth.expandable(),
+        maxHeightFunction: MaxHeight.expandable()
+      },
+      ...Coords.getPointAnchor(editor, e)
+    };
+  } else {
+    return Coords.getAnchorSpec(editor, e, anchorType);
+  }
+};
+
+const show = (editor: Editor, e: EditorEvent<TouchEvent>, items: MenuItems, backstage: UiFactoryBackstage, contextmenu: AlloyComponent, anchorType: Coords.ContextMenuAnchorType, highlightImmediately: boolean) => {
+
+  const anchorSpec = getAnchorSpec(editor, e, anchorType);
 
   NestedMenus.build(items, ItemResponse.CLOSE_ON_EXECUTE, backstage, true).map((menuData) => {
     e.preventDefault();
 
     // Show the context menu, with items set to close on click
+    const toolbarType = anchorType === 'node' ? 'node' : 'selection';
+    const contextToolbarBounds = getContextToolbarBounds(editor, backstage.shared, toolbarType);
```

Comment:
This change is incorrect... the bounds should be lazily computed (you'll note previously it was inside a thunk).

- [ ] Revert this change

Reply:
To add more as to why this is necessary, if the UI is repositioned (possibly due to a scroll or window resize) then it needs to check the bounds again which may have changed. As such the bounds can't be eagerly evaluated like this.